config/user-activity.php

```diff
@@ -7,9 +7,10 @@
     'delete_limit'     => 7,
 
     'log_events' => [
+        'on_create'    => true,
```

Comment:
- Default value `false` make it better.
- Write the value with proper indentation.

Reply:
Fixed on https://github.com/haruncpi/laravel-user-activity/pull/11/commits/5309dc43dee07f17960d6c3866fcec9779bf4877